### DISCUSSION TITLE
Hot Key Binds settings gone?

### DISCUSSION THREAD

**Discussion by BobbyPr** [BlitterStudio/amiberry#935] [Q&A] [answered]
using pi4b with retropie-buster-4.8-rpi_400.img (most current) with push buttons and joystick.
on initial setup going into retroarch > settings > input , i don't see the typical 'input hotkey binds" followed by the 'input user 1 binds" listing. Instead i see "hotkeys" followed by 'port 1 controls", 'port 2 controls' etc...  .
I've done new installs over this same pi before but had  this issue which is messing up quite a few things. Any ideas how to get back the "input user 1 binds" ?

**Comment by midwan** [comment, marked as answer]
This sounds like a question for the RetroPie team, not Amiberry. :)

**Reply by BobbyPr** [reply]
ok thanks
